sqlglot/expressions.py

```diff
@@ -1421,6 +1421,10 @@ class Constraint(Expression):
     arg_types = {"this": True, "expressions": True}
 
 
+class UnnamedConstraints(Expression):
+    arg_types = {"this": False, "expressions": True}
```

Comment:
FYI `this` is unused here. @Rik-de-Kort it seems like there might be some confusion around expression arguments, let me know if it's unclear and we can chat on Slack but in general `this` is not special so you don't need to add it everywhere.